draft-ietf-quic-transport.md

```diff
@@ -2314,33 +2320,31 @@ Therefore, a migrating endpoint can send to its peer knowing that the peer is
 willing to receive at the peer's current address. Thus an endpoint can migrate
 to a new local address without first validating the peer's address.
 
+To establish reachability on the new path, an endpoint initiates path
+validation ({{migrate-validate}}) on the new path.  An endpoint MAY defer path
+validation until after a peer sends the next non-probing frame to its new
+address.
```

Comment:
Why? Wouldn't that mean that there's a 1-RTT delay when the client switches to the new path, as the server now has to first verify the path?

Reply:
That's correct, it incurs a 1-RTT delay.  But it is a bit simpler to implement.

Reply:
This was moved from line 2324 FYI

Reply:
Yeah, I'd really hope that people _not_ do that and immediately validate in response (since state usage is still governed by how many CIDs they handed out) so that we can pre-warm the validation a few RTTs before we need to migrate. Perhaps we should call out that delay as a reason to avoid deferring unless the server is out of resources?

Reply:
Agreed. @ekinnear if you have text in mind, would you mind writing a PR to add your preferred clarification?